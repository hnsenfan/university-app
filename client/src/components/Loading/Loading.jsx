const Loading = () => (
  <label className='d-flex justify-content-center text-semibold text-18'>Loading...</label>
)

export default Loading