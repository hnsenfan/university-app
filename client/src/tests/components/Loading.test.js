import { shallow } from 'enzyme'
import Loading from '../../components/Loading/Loading'

test('should test Loading component', () => {
 const wrapper = shallow(<Loading />)
 expect(wrapper).toMatchSnapshot()
})