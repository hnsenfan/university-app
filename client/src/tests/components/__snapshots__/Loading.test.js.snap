// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Loading component 1`] = `
<label
  className="d-flex justify-content-center text-semibold text-18"
>
  Loading...
</label>
`;
